src/ff7/world/player.cpp

```diff
@@ -408,7 +408,9 @@ namespace ff7::world {
             {
                 if(player_model_id == HIGHWIND)
                     ff7_externals.world_run_special_opcode_7640BC(6);
-                else if(ff7_externals.world_is_player_model_bitmask(CLOUD | TIFA | CID | BUGGY) && ff7_externals.world_get_player_walkmap_type() != 14)
+                // For some reason function world_is_player_model_bitmask does not work so instead check the model id values manually
+                // else if(ff7_externals.world_is_player_model_bitmask(CLOUD | TIFA | CID | BUGGY) && ff7_externals.world_get_player_walkmap_type() != 14)
```

Comment:
Do we need the comment section above? I'd prefer having the fix directly with the reason explained in the PR if possible. Thanks!

Reply:
done